testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -540,7 +541,13 @@ public void testSimpleUnpartitionedTransactionalInsert()
         withTemporaryTable("unpartitioned_transactional_insert", true, false, NONE, tableName -> {
             onTrino().executeQuery(format("CREATE TABLE %s (column1 INT, column2 BIGINT) WITH (transactional = true)", tableName));
 
-            onTrino().executeQuery(format("INSERT INTO %s VALUES (11, 100), (12, 200), (13, 300)", tableName));
+            String insertQuery = format("INSERT INTO %s VALUES (11, 100), (12, 200), (13, 300)", tableName);
+
+            // ensure that we treat ACID tables as implicitly bucketed on INSERT
+            String explainOutput = (String) onTrino().executeQuery("EXPLAIN " + insertQuery).row(0).get(0);
+            Assertions.assertThat(explainOutput).contains("Output partitioning: hive:HivePartitioningHandle{buckets=1");
```

Comment:
nit: static import

Reply:
Can I have two static imports of method which is named the same? I though the same but IntelliJ did not suggest static import and I did not dig into that.